tiatoolbox/models/classification/patch_predictor.py

```diff
@@ -152,11 +152,9 @@ class CNNPatchPredictor:
         >>> data = [img1, img2]
```

Comment:
The Attributes list is not complete. I think you are missing `pretrained_models` and `pretrained_weights`.